<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
		background-color: rgb(139,26,219);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
		background-color: rgb(33,14,46);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_13">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="HomePage">
       <property name="styleSheet">
        <string notr="true">QWidget {
		background-color: rgb(33,14,46);
}



QLabel {
	color: rgb(139,26,219);
	background-color: rgb(33,14,46);
}
</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_load">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	color: rgb(139,26,219);
	background-color: rgb(33,14,46);
	font-size: 32px;
	border-radius: 20px;
	border: 2px solid rgb(139,26,219);
}

QPushButton:pressed {
	background-color: rgb(57,21,82);
}

QPushButton:hover {
	background-color: rgb(57,21,82);
	font-size: 33px;
}</string>
          </property>
          <property name="text">
           <string>   Выбрать и загрузить модель   </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_easteregg">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:72pt;&quot;&gt;3DViewer&lt;/span&gt;by tonitaga jerlenem&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ScenePage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="s21::ScenePaint" name="scene_widget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SceneSettingspage">
       <property name="styleSheet">
        <string notr="true">QWidget {
		background-color: rgb(33,14,46);
}

QLabel {
	color: rgb(139,26,219);
	background-color: rgb(33,14,46);
	font-size: 40 px;
}

QLineEdit {
	color: rgb(139,26,219);
	background-color: rgb(33,14,46);
	font-size: 32 px;
}

QComboBox {
	border: 1 px solid rgb(33,14,46);;
	color: rgb(139,26,219);
	background-color: rgb(33,14,46);
	font-size: 20 px;
}

QComboBox QAbstractItemView {
	border: 1 px solid rgb(33,14,46);;
	color: rgb(139,26,219);
	background-color: rgb(33,14,46);
	font-size: 20 px;
}

QPushButton {
	color: rgb(139,26,219);
	background-color: rgb(33,14,46);
	font-size: 20px;
	border-radius: 10px;
	border: 2px solid rgb(139,26,219);
}

QPushButton:pressed {
	background-color: rgb(57,21,82);
}

QPushButton:hover {
	background-color: rgb(57,21,82);
	font-size: 21px;
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_12">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="leftMargin">
           <number>180</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>170</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <property name="horizontalSpacing">
           <number>130</number>
          </property>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboBox_vertex_size">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="currentIndex">
             <number>5</number>
            </property>
            <item>
             <property name="text">
              <string>0.1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="combobox_projection">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Параллельная</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Центральная</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton_accept">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Подтвердить все</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_const_3">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Цвет ребра</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pushButton_vertex_color">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Выбрать цвет</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_const_5">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Тип ребра</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBox_rib_type">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <item>
             <property name="text">
              <string>Сплошной</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Пунктирный</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="comboBox_showness">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Нет</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Круглые</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Квадратные</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_const_1">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	color: rgb(139,26,219);
	background-color: rgb(33,14,46);
	font-size: 32 px;
}</string>
            </property>
            <property name="text">
             <string>Ширина ребра</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_rib_size">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>0.1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3.5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_rib_color">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Выбрать цвет</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_const_4">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Размер вершин</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_const_6">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Цвет вершины</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_const_0">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Отображение</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_const_2">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Вид точек</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="pushButton_bg_color">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Выбрать цвет</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_const_7">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Цвет фона</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_mainwinow"/>
    <addaction name="action_scene"/>
    <addaction name="action_modelinfo"/>
    <addaction name="action_scenesettings"/>
    <addaction name="action_imagemodel"/>
    <addaction name="action_gifmodel"/>
    <addaction name="action_aboutprogram"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_mainwinow">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/3dViewer.ico</normaloff>:/Icons/3dViewer.ico</iconset>
   </property>
   <property name="text">
    <string>Главное окно</string>
   </property>
  </action>
  <action name="action_scene">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/scene.ico</normaloff>:/Icons/scene.ico</iconset>
   </property>
   <property name="text">
    <string>Сцена</string>
   </property>
  </action>
  <action name="action_modelinfo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/info.ico</normaloff>:/Icons/info.ico</iconset>
   </property>
   <property name="text">
    <string>Информация о модели</string>
   </property>
  </action>
  <action name="action_scenesettings">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/settings.ico</normaloff>:/Icons/settings.ico</iconset>
   </property>
   <property name="text">
    <string>Настройки сцены</string>
   </property>
  </action>
  <action name="action_aboutprogram">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/aboutico.ico</normaloff>:/Icons/aboutico.ico</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_imagemodel">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/jpg.ico</normaloff>:/Icons/jpg.ico</iconset>
   </property>
   <property name="text">
    <string>Снимок модели</string>
   </property>
  </action>
  <action name="action_gifmodel">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/gif.ico</normaloff>:/Icons/gif.ico</iconset>
   </property>
   <property name="text">
    <string>Гифка модели</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>s21::ScenePaint</class>
   <extends>QOpenGLWidget</extends>
   <header>../Scene/scene_paint.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
